draft-js/draft-js-tests-0.2.2.tsx

```diff
@@ -0,0 +1,183 @@
+/// <reference path="draft-js-0.2.2.d.ts" />
+/// <reference path="../react/react-global.d.ts" />
+/// <reference path="../react/react-dom.d.ts"/>
+
+import * as React from "react";
+
+import {
+Editor,
```

Comment:
Hi @dmitryrogozhny I understand that, I am simply saying there is a formatting issue with the indentation on this line. What I saw is:
![capture](https://cloud.githubusercontent.com/assets/1171301/16743300/77eea8f2-4760-11e6-9f88-db1ab7ba0053.PNG)
It should be 
```
import {
    Editor,
    EditorState
    ...
```
instead of 
```
import  {
Editor,
    EditorState
    ...
```